core/common.ts

```diff
@@ -7,7 +7,12 @@
 // Former goog.module ID: Blockly.common
 
 import type {Block} from './block.js';
-import {ISelectable} from './blockly.js';
+import {
+  BlockSvg,
+  ISelectable,
+  getFocusManager,
+  isSelectable,
+} from './blockly.js';
```

Comment:
These probably shouldn’t be imported from blockly.js but from their real files. Vs code will do the wrong thing sometimes unfortunately.

Reply:
Yep, this was definitely VS Code in a bunch of places. I've fixed this here, and updated elsewhere to use more `import type` since it largely seems better for reducing circular dependency risks.

Reply:
This has been addressed--marking as resolved to keep open only those threads that may need additional discussion.